WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationFormViewModel.swift

```diff
@@ -62,21 +62,21 @@ final class BlazeCampaignCreationFormViewModel: ObservableObject {
     }
 
     /// We need to recreate the view model every time the budget screen is opened to get the updated target options.
-    var budgetSettingViewModel: BlazeBudgetSettingViewModel {
+    lazy private(set) var budgetSettingViewModel: BlazeBudgetSettingViewModel = {
```

Comment:
Making the view model lazy is important to not recreating it every time it's accessed. This helps with unit tests as well.